BenchPress/Helpers/Azure/Common.psm1

```diff
@@ -22,13 +22,52 @@ enum ResourceType {
   WebApp
 }
 
-function Get-ResourceByType {
+<#
+.SYNOPSIS
+  Gets an Azure Resource.
+
+.DESCRIPTION
+  The Get-AzBPResourceByType cmdlet gets an Azure resource depending on the resource type (i.e. Action Group, Key Vault,
+  Container Registry, etc.).
+
+.PARAMETER ResourceName
+  The name of the Resource
+
+.PARAMETER ResourceGroupName
+  The name of the Resource Group
+
+.PARAMETER ResourceType
+  The type of the Resource (currently support the following:
+  ActionGroup
+  AKSCluster
+  AppServicePlan
+  ContainerRegistry
+  KeyVault
+  ResourceGroup
+  SqlDatabase
+  SqlServer
+  VirtualMachine
+  WebApp)
+
+.EXAMPLE
+  Get-AzBPResourceByType -ResourceType ActionGroup -ResourceName "bpactiongroup" -ResourceGroupName "rgbenchpresstest"
+
+.EXAMPLE
+  Get-AzBPResourceByType -ResourceType VirtualMachine -ResourceName "testvm" -ResourceGroupName "rgbenchpresstest"
+
+.INPUTS
+  System.String
+
+.OUTPUTS
+  Microsoft.Azure.Commands.ResourceManager.Cmdlets.SdkModels.PSResource
+#>
+function Get-AzBPResourceByType {
```

Comment:
With the prefix being added will this result in `Get-AzBPAzBPResourceType`? Or is the prefixing smart enough to know that it already exists?

Reply:
Didn't mean to add the prefix here. Reverting to original name.